react/features/large-video/components/LargeVideo.native.js

```diff
@@ -130,7 +131,7 @@ class LargeVideo extends PureComponent<Props, State> {
                 disableVideo = { _disableVideo }
                 onPress = { onClick }
                 participantId = { _participantId }
-                style = { _styles.largeVideo }
+                style = { [ _styles.largeVideo ] }
```

Comment:
It didn't quite understand how adding the styles object in a single item array helped the backgroundColor and backgroundImageUrl get applied on the ParticipantView also.

Reply:
What? That's odd indeed. Can you re-test?